src/adagrams.js

```diff
@@ -1,15 +1,144 @@
+const letterPool = {
```

Comment:
We can name variables that are constant values (meaning that they have values that are hardcoded) with capital letters and snake_case, so letterPool could be renamed LETTER_POOL. 

Also, in the future, a way to organize code is creating another file like constants.js and you could put LETTER_POOL in there and import it into adagrams.js to use. Totally optional, but sometimes folks like to pull big objects into another file to keep their main javascript files more concise. Here's an [example](https://stackoverflow.com/questions/61213322/how-to-import-object-to-another-file/61755282#61755282) of how to do it.